config_holesky.py

```diff
@@ -0,0 +1,8 @@
+lido_dao_agent_address = "0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d"
+lido_easy_track_script_executor_address = "0x2819B65021E13CEEB9AC33E77DB32c7e64e7520D"
+
+dai_token_address = None
+dai_token_holder_address = None
+
+usdt_token_address = None
+usdt_token_holder_address = None
```

Comment:
I think a better practice would be to capitalize these variable names, so that whenever we stumble upon them in other files, we can at once see these are constant values taken from config.